common/culture/traditions/00_regional_traditions.txt

```diff
@@ -88,15 +102,9 @@ tradition_lords_of_the_elephant = {
 			has_innovation = innovation_war_dinosaur
 		}
 	}
-	can_pick = {
-		OR = {
-			has_innovation = innovation_elephantry
-			has_innovation = innovation_war_dinosaur
-		}
-	}
 	
 	parameters = {
-		can_recruit_jungle_specialist = yes
+		can_recruit_open_specialist = yes
```

Comment:
Could you explain the idea?

Reply:
The implementation in `1.5-preview-adaptation` redesigned the tradition for Zandalari from looking at the triggers and the added parameter `can_recruit_jungle_specialist = yes`. However, since Elephants are now a regional maa and also used by Draenei (tradition_lords_of_the_elephant_desc and tradition_lords_of_the_elephant_regional_desc) the tradition should confer a more neutral terrain bonus instead of one specific to the Jungle. Large beasts like Elephants or Dinosaurs should excel as heavy cavalry in the open plains either way.

Reply:
Mmmkay, but this bonus doesn't work for dinos then.

Reply:
I mean we also have this tradition for dino MaA. Their home region is a jungle. Probably need to split this tradition in 2. This one and another one for dinos.

Reply:
To be addressed in #981